.github/workflows/pull-request.yml

```
@@ -0,0 +1,265 @@
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'
+
+      - name: Install Dependencies
+        run: npm install
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)